src/lib/DocsSearchBar.js

```diff
@@ -158,6 +148,23 @@ class DocsSearchBar {
     }
   }
 
+  static addThemeWrapper(inputSelector, enableDarkMode) {
+    const inputElement = document.getElementById(inputSelector.substring(1))
+    const parent = inputElement.parentNode
+    const wrapper = document.createElement('div')
```

Comment:
```suggestion
    const wrapper = document.createElement('div')
    wrapper.className += 'docs-searchbar-js'
```

Add default class in case there is no `data-ds-theme`  so the user can still access this div